### ISSUE TITLE
archivos .H5

### ISSUE THREAD

**Issue by RAnastacioo** [Machine-Listeners-Valencia/DCASE2021-Task1b#4]
Felicitaciones por su participación en el concurso.
Me gustaría aprender a crear los archivos .h5 que usa para entrenar y evaluar su red neuronal.
Creo que este código no está en el repositorio.

Otra duda los datos que contiene el archivo .h5 fueron normalizados ??

Otra cuestión
La baseline está disponible en pytorch pero me gustaría usar tensorflow.
¿Es posible reproducir fielmente la baseline en tensorflow?

**Comment by JNaranjo-Alcazar** [comment]
The repo has been updated with all necessary information. If any question, please create a new issue :)
